mathesar_ui/README.md

```diff
@@ -63,6 +63,22 @@ node@c273da65c52d:/ui$ npm install <package>
 node@c273da65c52d:/ui$ npm uninstall <package>
 ```
 
+### Components
+
+For guidelines on component development, refer [README of components](https://github.com/centerofci/mathesar/tree/integrate_storybook/mathesar_ui/src/components#readme).
+
+You can start storybook in dev mode by connecting to the container and then running `npm run storybook`:
+
+```bash
+docker exec -it mathesar_ui_1 /bin/bash
+
+node@c273da65c52d:/ui$ npm run storybook
```

Comment:
These instructions are not working for me.

```
node@62013923826d:~/ui$ npm run storybook

> mathesar_ui@0.0.0 storybook /home/node/ui
> start-storybook -p 6006

sh: 1: start-storybook: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! mathesar_ui@0.0.0 storybook: `start-storybook -p 6006`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 storybook script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-06-30T19_58_17_245Z-debug.log
```

Reply:
This could be an issue with the Storybook bins not present in `node_modules/`. Try by rebuilding the container?

Reply:
This issue is because storybook was not installed. Restarting the container would perform an `npm install`, and should install the required packages.